cmd/otel-allocator/README.md

```diff
@@ -189,6 +189,29 @@ The CRs can be filtered by labels as documented here: [api/opentelemetrycollecto
 
 Upstream documentation here: [PrometheusReceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver#opentelemetry-operator)
 
+### Pod/Service Monitor Selectors
+
+As of `v1beta1` of the OpenTelemetryOperator, a `serviceMonitorSelector` and `podMonitorSelector` must be included, even if you don’t intend to use it, like this:
+
+```yaml
+prometheusCR:
+  enabled: true
+  podMonitorSelector: {}
+  serviceMonitorSelector: {}
```

Comment:
I believe if you are using the targetAllocator section in the collector we default this, but def true here that you need to set it.

Reply:
hum, weird. I had to explicitly set it since the Allocator was not finding the ServiceMonitors, then I found the troubleshooting tips 😀